### DISCUSSION TITLE
How to preview remix application on mobile ?

### DISCUSSION THREAD

**Discussion by ishub-mobi** [remix-run/remix#9912] [Q&A] [answered]
I want to preview the dev build on mobile, to fix some mobile specific issues.
tried using ngrok package for tunneling, but in this the CSS is not loading, only html is rendering.

I'm using tailwind CSS, and I have some custom CSS as well.

**Comment by sergiodxa** [comment, marked as answer]
In Vite, if you add `--host` it will give you the IP address, this is part of Vite not Remix.
